### ISSUE TITLE
Lens profile selection not available

### ISSUE THREAD

**Issue by MareskoY** [WeAreNewt/re-meme#145]
Steps:
1. clean up cash local storage and etc. and open the app
2. connect the wallet
3. choose a Lens profile
4. disconnect
5. connect again

Result:
- choosing a profile  is not available

![Sep-06-2022 11-18-20](https://user-images.githubusercontent.com/18032275/188611479-39f4bae9-2d2a-4f55-8d1b-b298e0272fc5.gif)

**Comment by 0xjim** [comment]
You're right. In this workflow, user has to refresh the page and sign again before the Lens Profile is detected.
